src/commands/base-command.js

```diff
@@ -470,7 +470,10 @@ class BaseCommand extends Command {
       return await resolveConfig({
         config: options.config,
         cwd,
-        context: options.context || this.name(),
+        context:
+          options.context || process.env.CONTEXT || ['dev', 'dev:exec', 'dev:trace'].includes(this.name())
```

Comment:
This was done in the child dev command classes when we had `oclif`. Not sure if there's a better way to do it now.